editor/animation_track_editor.h

```diff
@@ -41,6 +41,8 @@
 #include "scene/gui/tree.h"
 #include "scene/resources/animation.h"
 
+#define SCROLL_ZOOM_FACTOR 1.02 // Zoom factor per mouse scroll in the animation editor. The closer to 1.0, the finer the control.
```

Comment:
This should be defined as `static constexpr float`, preferably inside the class.

Reply:
moved it down to AnimationTimelineEdit, I think that's where it belongs the most